_posts/2018-05-30-cockpit-169.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Cockpit 169
+author: pitti
+date: 2018-05-30
+tags: cockpit linux
+slug: cockpit-169
+category: release
+summary: Cockpit with NFS, FreeIPA, and systemd improvements
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We release regularly.
+Here are the release notes from version 169.
+
+### Storage: Offer installation of NFS client support
+
+The "NFS Mounts" pane on the Storage page now shows an "Install NFS Support"
+button if the necessary packages are not already available:
+
+<iframe width="960" height="720" src="https://www.youtube.com/embed/yR5riNcIDbc?rel=0" frameborder="0" allowfullscreen></iframe>
+
+### System: Request FreeIPA SSL certificate for Cockpit's web server
+
+When enrolling into a [FreeIPA](https://freeipa.org/) domain, an SSL
+certificate is requested from the IPA server and stored in
+`/etc/cockpit/ws-certs.d/10-ipa.cert`. Cockpit's web server will then use that
+certificate instead of the default self-signed one.  The IPA server's
+certificate authority is trusted by default and accepted by web browsers,
+OpenSSL, curl, and other network clients.
+
+### Services: Show unit relationships
+
+The details of a systemd service now shows dependency, ordering, and other
+relationships to units:
+
+![Services relationships](/images/services-relationships.png)
+
+### Provide motd help how to access cockpit
```

Comment:
Normally, I'd say `motd` should be capitalized as MOTD, as it stands for "message of the day"... but I checked Wikipedia and it's lowecase. So, I guess it's referring to `/etc/motd`, where it is lowercase. Since that's the actual name of the file, I suppose `motd` is correct, instead of `MOTD` in this case?

**Summary**: OK as-is.

Reply:
Right, the file, manpage, and general concept are usually written "motd", I've never seen "MOTD". Hysterical raisins.. :-)